Gemfile.lock

```diff
@@ -249,8 +249,8 @@ GEM
     naturally (2.2.1)
     netrc (0.11.0)
     no_proxy_fix (0.1.2)
-    nokogiri (1.11.5)
-      mini_portile2 (~> 2.5.0)
+    nokogiri (1.13.8)
```

Comment:
@AleksanderWedrychowskiKarhoo  @BartlomiejSopalaKarhoo 1.13.6 had some security issues which made us revert to 1.11.5. I vote we close this and bump directly to 1.14 when it's available. Thoughts?